src/Sulu/Bundle/ContentBundle/Tests/Features/content_management.feature

```diff
@@ -36,6 +36,7 @@ Feature: Content management
         And I fill in "body" with "Dornbirn is great"
         And I click the save icon
         And I click toolbar item "savePublish"
+        And I confirm
```

Comment:
All these `And I confirm` are necessary because after clicking on publish an overlay appears now.